apps/ledger-live-mobile/src/screens/SyncOnboarding/SoftwareChecksStep.tsx

```diff
@@ -190,15 +187,11 @@ const SoftwareChecksStep = ({ device, isDisplayed, onComplete }: Props) => {
 
         // Updates the genuineCheckStatus
         if (genuineCheckError) {
-          console.error(
-            "Failed to perform genuine check with error:",
-            genuineCheckError.message,
-          );
           setGenuineCheckStatus("failed");
         } else if (genuineState === "genuine") {
           setGenuineCheckStatus("completed");
         } else if (genuineState === "non-genuine") {
-          // FIXME: if the device is non-genuine, we should display something else
+          nextDrawerToDisplay = "cancelled";
           setGenuineCheckStatus("failed");
```

Comment:
Ok actually: (i removed some comments i've put): we are keeping this `setGenuineCheckStatus("failed");` here, but can you remove the `nextDrawerToDisplay = "cancelled";` you've added ?

Here we want to only update the `genuineCheckStatus` (to keep a bit of organization in the code), see the comments just above the first `if` of this block

The `nextDrawerToDisplay` is set correctly just after where you added:
```
} else if (genuineCheckStatus === "failed") {
        nextDrawerToDisplay = "cancelled";
```